src/main/resources/linux64/goobox-sync-storj

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+java -Djava.library.path=. -jar goobox-sync-storj-0.0.26.jar
```

Comment:
Why don't you reuse the existing `src/main/resources/goobox-sync-storj`? Should work for both Linux and macOS.

Reply:
did that before, ask about it (to meije or you few weeks back?) the error still the following

```
$ java -Djava.library.path=. -jar goobox-sync-storj-0.0.26.jar
[main] DEBUG net.harawata.appdirs.AppDirsFactory - os.name linux is resolved to *nix
SLF4J: A number (1) of logging calls during the initialization phase have been intercepted and are
SLF4J: now being replayed. These are subject to the filtering rules of the underlying logging system.
SLF4J: See also http://www.slf4j.org/codes.html#replay
Exception in thread "main" java.lang.UnsatisfiedLinkError: no storj-java in java.library.path
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867)
	at java.lang.Runtime.loadLibrary0(Runtime.java:870)
	at java.lang.System.loadLibrary(System.java:1122)
	at io.storj.libstorj.Storj.loadLibrary(Storj.java:425)
	at io.storj.libstorj.Storj.<clinit>(Storj.java:409)
	at io.goobox.sync.storj.App.init(App.java:150)
	at io.goobox.sync.storj.App.main(App.java:106)
```

Reply:
@jkawamoto will adding `-Djava.library.path=.` have negative effect on macOS?

Reply:
I think it’s okay to set `java.library.path` on macOS. 

Btw, could you add `$GOOBOX_SYNC_STORJ_OPTS` before `-jar` so that the GUI can pass the paths of overlay icons using the environment variable?

Reply:
sure